packages/legacy-client/src/client/ontology.test.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createThinClient, type ThinClient } from "@osdk/api";
+import type { MockedFunction } from "vitest";
+import { assert, beforeEach, describe, expect, it, vi } from "vitest";
+import { ActionExecutionMode, ReturnEditsMode } from "../ontology-runtime";
+import {
+  expectFetchToBeCalledWithBody,
+  expectFetchToBeCalledWithGet,
+  mockFetchResponse,
+} from "../util/test/fetchUtils";
+import { MockOntology } from "../util/test/mockOntology";
+import { MOCK_ORIGIN } from "../util/test/mocks/mockMetadata";
+import { mockTodoObject } from "../util/test/mocks/mockObjects";
+import { unwrapResultOrThrow } from "../util/test/resultUtils";
+import { Ontology } from ".";
+
+describe(Ontology, () => {
```

Comment:
```suggestion
describe("Ontology", () => {
```

Reply:
I saw this used a few places, in vitest it shows the right name, don't feel super strong about this though

Reply:
The examples from vitest use strings here, I'll add it to the backlog to move everything over to quoted strings. For some reason it accepts Functions as well.